donate.html

```diff
@@ -65,9 +65,68 @@
 
 </head>
 <body>
+
 <div class="container-fluid">
   <!-- su'ad -->
-
+  <!-- Header -->
+  <div class = "donate-header">
+    <div style="text-align:center;">
+      <h2>
+        <span style="position: relative">
+        Donating to
+        <div class="d-none d-md-block">
+          <div class="large-spark pink-spark" style="top: -30px; left: -70px;"></div>
+          <div class="med-spark purp-spark" style="top: 30px; left: -120px;"></div>
+          <div class="small-spark blue-spark" style="top: 120px; left: -50px;"></div>
+          <div class="large-spark purp-spark" style="top: -35px; right: -35px;"></div>
+          <div class="small-spark pink-spark" style="top: 150px; right: -25px;"></div>
+          <div class="med-spark blue-spark" style="top: 70px; right: -110px;"></div>
+        </div>
+      </span>
+    </h2>
+      <div> <img src="img/black_colorstripe.png" width="270" height="60"></div>
+      <div class="b1"> <button>Donate</button> </div>
+    </div>
+  </div>
+  <!-- Our Story  -->
+  <div class="donate-story">
+    <div class="story-header">
+      <h1>Our Story</h1>
+    </div>
+    <div class="row">
+      <div class="col-md-6">
+        <div class="story-body">
+          <p> Technica was founded in 2015 by empowered students at the University of Maryland who recognized the need for an inclusive all-women & non-binary space in the hackathon scene. </p>
+          <p> As the world’s largest all-women & non-binary hackathon, we are often asked how one can support our cause. With time constraints and distance in mind, we’ve created a way for individuals to offer support: financial donations. </p>
+          <p> We know money can be tight, so any and all donations are greatly appreciated. Pitching in a few bucks towards our cause means we can continue offering the highest quality experience for our hackers. </p>
+        </div>
+      </div>
+      <div class="col-md-6">
+        <div class="story-image">
+          <img src="img/2019_img.jpg" class="center-block" width="350" height="200">
+        </div>
+      </div>
+    </div>
+  </div>
+  <!-- Donor Promise -->
+  <div class="donate-promise">
+    <!--style = "position:relative; left:80px; top:2px;" for the header -->
+    <div class="promise-header">
```

Comment:
this heading should be lined up with the paragraph below, a hack to get this effect is to create another row here, have an empty col-6 followed by another col-6 with "Our Donor Promise" in it